REFERENCE.md

```diff
@@ -7,6 +7,7 @@
 ### Classes
 
 * [`profile_acctd`](#profile_acctd): Manage resources related to NCSA acctd.
+* [`profile_acctd::backup`](#profile_acctd--backup): Configure xCAT master backups
```

Comment:
REFERENCE.md still needs to be run again...
`pdk bundle exec puppet strings generate --format markdown`

Reply:
```
kingda@Porsche:~/working/puppet-profile_acctd$ pdk bundle exec puppet strings generate --format markdown
pdk (INFO): Using Ruby 2.7.8
pdk (INFO): Using Puppet 7.25.0
Files                     2
Modules                   0 (0 undocumented)
Classes                   0 (0 undocumented)
Constants                 0 (0 undocumented)
Attributes                0 (0 undocumented)
Methods                   0 (0 undocumented)
Puppet Data Type Aliases  0 (0 undocumented)
Puppet Classes            2 (0 undocumented)
Puppet Types              0 (0 undocumented)
Puppet Data Types         0 (0 undocumented)
Puppet Functions          0 (0 undocumented)
Puppet Defined Types      0 (0 undocumented)
Puppet Providers          0 (0 undocumented)
Puppet Tasks              0 (0 undocumented)
Puppet Plans              0 (0 undocumented)
 100.00% documente
```